src/TailscaleManager.hs

```diff
@@ -120,7 +119,7 @@ runOnce options prevRoutes = do
           logL logger DEBUG ("Sleeping for " ++ show (interval options) ++ " seconds")
         threadDelay (interval options * 1000000)  -- microseconds
 
-  config <- loadConfig (configFile options)
+  config <- TailscaleManager.Config.loadConfig (configFile options)
```

Comment:
I don't think this name qualification is required; ghc would be complaining if it was ambiguous.

Reply:
sure, resolved :)